egs/thchs30/s5/local/insert_blank.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python2.6
```

Comment:
I suggest using only shebang
#!/usr/bin/env python

because I believe your program should be able to run in pythin 2.7 as well.